ftw/upgrade/workflow.py

```diff
@@ -126,18 +126,23 @@ def _migrate_workflow_history(self, context, old_wf, new_wf):
         if wfhistory is None or old_wf not in wfhistory:
             return
 
-        def _migrate_action(entry):
+        def _migrate_transition(entry):
```

Comment:
I think `action` is correct; there might be entries which may not be transitions but additional information (such as file downloaded in ftw.file).
I'd keep the `_migrate_action` name and make sure that those entries are working too.